makefiles/utils/test-variables.mk

```diff
@@ -3,7 +3,7 @@ include variables.mk
 # Timestamp in nanoseconds (to be an integer)
 # OSx 'date' does not support 'date +%s%N' so rely on python instead
 # It could be OSx specific but we do not have 'OS' defined here to differentiate
-date_nanoseconds = $(shell python -c 'import time; print(int(time.time() * 1000000000))')
+date_nanoseconds = $(shell python3 -c 'import time; print(int(time.time() * 1000000000))')
```

Comment:
Is python3 more universal? I'd always expect python to be there but not python3?`

Reply:
Never took french in school so I can only respond with

"Oepsie doepsie, hoe meer je weet"